sender_integration_test.go

```diff
@@ -236,7 +238,7 @@ func TestE2EValidWrites(t *testing.T) {
 					Symbol("sym_col", "test_ilp1").
 					Float64Column("double_col", 12.2).
 					Int64Column("long_col", 12).
-					Long256Column("long256_col", "0x123a4i").
+					Long256Column("long256_col", big.NewInt(0).SetInt64(0x123a4)).
```

Comment:
It's better to use [this method](https://pkg.go.dev/math/big#Int.SetString) to keep the code more human-readable.

Reply:
I didn't use `SetString()` because returns two arguemnt, which does not fit there.
Example:
```go
	// Method 1
    // If you inisit I can declare a variable above in the test and use it again to assign differnt values like this.
    var bigVal big.Int
	bigVal.SetString("FF", 16)

    // Method 2
    // this would be invalid, since it returns two values.
    Long256Column("long256_col", big.NewInt(0).SetString("FF", 16)
```
So I can make a few changes and customise the tests similar to method 1. Also apply the same pattern below.

Reply:
There is nothing wrong with adding a few lines to initialize the `big.Int` separately.